webui/react/src/hooks/usePermissions.ts

```diff
@@ -289,6 +292,15 @@ const canDeleteModelVersion = (
 
 // Project actions
 // Currently the smallest scope is workspace
+const canCreateProject = (
+  workspace?: PermissionWorkspace,
+  userAssignments?: UserAssignment[],
+  userRoles?: UserRole[],
+): boolean => {
+  const permitted = relevantPermissions(userAssignments, userRoles, workspace?.id);
+  return permitted.has('oss_user') || permitted.has('create_project');
```

Comment:
Since the API isn't going to return this after all, we should be using the feature switch here.